src/imputors/interp.jl

```diff
@@ -3,30 +3,30 @@
 
 Performs linear interpolation between the nearest values in an vector.
 """
-type Interpolate <: Imputor end
+struct Interpolate <: Imputor end
 
 """
-    impute!{T<:Any}(imp::Interpolate, ctx::Context, data::AbstractArray{T, 1})
+    impute!(imp::Interpolate, ctx::Context, data::AbstractVector)
 
 Uses linear interpolation between existing elements of a vector to fill in missing data.
 
 WARNING: Missing values at the head or tail of the array cannot be interpolated if there
 are no existing values on both sides. As a result, this method does not guarantee
 that all missing values will be imputed.
 """
-function impute!{T<:Any}(imp::Interpolate, ctx::Context, data::AbstractArray{T, 1})
+function impute!(imp::Interpolate, ctx::Context, data::AbstractVector{T}) where T
```

Comment:
You can make this
```suggestion
function impute!(imp::Interpolate, ctx::Context, data::AbstractVector{<:Union{T,Missing}}) where T
```
then drop the `Missings.T` later on, as well as the dependency on Missings altogether.